src/cpp/transport/tcp/RTCPMessageManager.cpp

```diff
@@ -258,11 +268,12 @@ void RTCPMessageManager::fillHeaders(
     */
 }
 
-TCPTransactionId RTCPMessageManager::sendConnectionRequest(TCPChannelResource *p_channel_resource)
+TCPTransactionId RTCPMessageManager::sendConnectionRequest(
+        std::shared_ptr<TCPChannelResource>& channel)
```

Comment:
Using a reference instead of a copy can cause problems as the channel being freed while we are still using it.
Apply coding style (one argument, one line method).